includes/text-teaser.html

```diff
@@ -4,26 +4,13 @@
 {%- assign textTeaserCopy = include.copy -%}
 {%- assign textTeaserList = include.list -%}
```

Comment:
Könnte man dann auch schon textTeaserCtaList nennen und vielleicht in der Zukunft die Stellen, an denen die Komponente verwendet wird refactorn.